spec/features/users_can_export_spending_breakdown_spec.rb

```diff
@@ -16,6 +18,23 @@
       expect(page).to have_content(export_in_progress_msg)
     end
 
+    context "when a fund has an uploaded spending breakdown" do
+      let(:newton_fund) { Fund.by_short_name("NF").activity }
+
+      before do
+        newton_fund.update!(spending_breakdown_filename: "spending_breakdown-20230130120000.csv")
+      end
+
+      scenario "they can download the existing spending breakdown as well as request a new one" do
+        visit exports_path
+
+        newton_fund_id = Fund.by_short_name("NF").id
+        expect(page).to have_link("Download Spending breakdown for Newton Fund", href: spending_breakdown_download_export_path(newton_fund_id))
+        expect(page).to have_content("last generated at 2023-01-30 12:00")
+        expect(page).to have_link("Request new Spending breakdown for Newton Fund", href: spending_breakdown_exports_path(fund_id: newton_fund_id))
```

Comment:
Can we lower case the "S"s in "Download Spending" and "Request new Spending"?

Reply:
The full text is only shown to screen readers to give context to the actions, but I take your point. I'll see if I can squeeze it in.

Reply:
The same translation is being used in the (visible) table cell for the export's type and name, so I'm inclined to leave as is rather than create duplicate lowercase translation strings.

Reply:
Ah, if it's just for screen readers, no need to worry too much about style!